servers/physics_3d/godot_body_3d.cpp

```diff
@@ -472,6 +472,13 @@ bool GodotBody3D::is_axis_locked(PhysicsServer3D::BodyAxis p_axis) const {
 	return locked_axis & p_axis;
 }
 
+void GodotBody3D::offset_position_for_ccd(const Vector3& offset) {
+	Transform3D transform = get_transform();
```

Comment:
```suggestion
	Transform3D tmp_transform = get_transform();
```
Or similar, showing a member variable